src/views/blog/Blog.query.ts

```diff
@@ -29,7 +29,7 @@ type QueryResult = {
 export const useBlogQuery = () => {
   const result: QueryResult = useStaticQuery(graphql`
     query BlogPage {
-      markdownRemark(fileAbsolutePath: { regex: "/blog/index-1.md/" }) {
+      markdownRemark(fileAbsolutePath: { regex: "/blog/index.md/" }) {
```

Comment:
What does this change do 🤔 ?

Reply:
When I added the blog again it generated an `index.md` file instead of `index-1.md`. Without this change blog would break the app